NETProvider/src/FirebirdSql.Data.FirebirdClient/Client/Managed/Version10/GdsConnection.cs

```diff
@@ -140,24 +161,24 @@ public virtual void Identify(string database)
 			{
 				outputStream.Write(IscCodes.op_connect);
 				outputStream.Write(IscCodes.op_attach);
-				outputStream.Write(IscCodes.CONNECT_VERSION2);	// CONNECT_VERSION2
+				outputStream.Write(IscCodes.CONNECT_VERSION3);	// CONNECT_VERSION2
 				outputStream.Write(1);							// Architecture	of client -	Generic
 
 				outputStream.Write(database);					// Database	path
-				outputStream.Write(3);							// Protocol	versions understood
+				outputStream.Write(4);							// Protocol	versions understood
 				outputStream.WriteBuffer(UserIdentificationStuff());	// User	identification Stuff
 
 				outputStream.Write(IscCodes.PROTOCOL_VERSION10);//	Protocol version
 				outputStream.Write(1);							// Architecture	of client -	Generic
 				outputStream.Write(2);							// Minimum type (ptype_rpc)
 				outputStream.Write(3);							// Maximum type (ptype_batch_send)
-				outputStream.Write(0);							// Preference weight
+				outputStream.Write(1);							// Preference weight
```

Comment:
Oh, sorry, I'm confuse about Preference weight.
revert and fix about Preference weight.